zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -969,4 +953,15 @@ public static void setReconfigEnabled(boolean enabled) {
         reconfigEnabled = enabled;
     }
 
+    private boolean parseBoolean(String value) throws ConfigException {
+        if (value.equalsIgnoreCase("true")) {
+            return true;
+        } else if (value.equalsIgnoreCase("false")) {
+            return false;
+        } else {
+            throw new ConfigException("Invalid option "
```

Comment:
please also log the parameter name, helping the users to find out which config is wrong.